app/models/scholarship_application.rb

```diff
@@ -1,17 +1,9 @@
 class ScholarshipApplication < ApplicationRecord
-  validate :user_is_verified
   validates :user_id, uniqueness: {scope: :scholarship_id}
 
   belongs_to :user
   belongs_to :scholarship
 
   delegate :name, to: :user, prefix: true
   delegate :email, to: :user, prefix: true
-  delegate :verified, to: :user, prefix: true
```

Comment:
This tells me there might be a method in the user model related to a user being verified - can you check whether there is one there that should be removed as well?

Reply:
https://github.com/OperationCode/operationcode_backend/blob/5c3152374851db4a8607b73a0dd66e0e85571a98/app/models/user.rb#L9

```ruby
 LEADER = 'community leader'

  # Validation of military_status
  CURRENT = 'current'
  VETERAN = 'veteran'
  SPOUSE = 'spouse'
  MILITARY_STATUSES = [CURRENT, VETERAN, SPOUSE, nil]
  validates :military_status, inclusion: { in: MILITARY_STATUSES }
```

This is what I see for those. I wasn't sure to remove them as I don't know how these affect the current database.

Reply:
Yup they ran locally and travis ran well too. 

```powershell
Finished in 24.010227s, 7.7467 runs/s, 25.4058 assertions/s.

186 runs, 610 assertions, 0 failures, 0 errors, 2 skips

You have skipped tests. Run with --verbose for details.
Inspecting 251 files
...........................................................................................................................................................................................................................................................

251 files inspected, no offenses detected
C:\Users\wimo7\Desktop\back [cancel_id_me +1 ~0 -0 !]>
```